handle headers for get requests too

massivefermion opened the pull request:
#1821 solved the header problem when running `platformatic client <url> --frontend` for all http methods except for get, so this is the fix.

massivefermion commented:
Sorry for the mess, apparently I don't know how to change my commit message to add the DCO!

mcollina reviewed (approved):
lgtm

mcollina commented:
if you click on the DCO action there are the instructions ;)

massivefermion commented:
Sorry for so many commits, I saw that I forgot to close the parenthesis for the fetch call, fixed that on github, and then realized I forgot the DCO on that commit! So I just had to amend on my laptop!

mcollina commented:
There seem to be a relevant failing test https://github.com/platformatic/platformatic/actions/runs/6990199433/job/19052799895?pr=1842#step:7:407

massivefermion commented:
Honestly, I don't understand what's going on. I literally just copied the `handle headers parameters` test above it and changed method to `GET`, removed the body and the `Content-type` header.
I also put a line in the test to copy the files generated by both tests somewhere else so I could check them and they're similar(except for the above changes).
So it seems to be working as it's expected but I don't understand why my test is failing.

massivefermion commented:
It's fixed now!
